iaso/api/mobile/entity.py

```diff
@@ -75,8 +75,14 @@ class Meta:
     org_unit_id = serializers.CharField(read_only=True, source="org_unit.id")
     form_version_id = serializers.SerializerMethodField()
 
-    created_at = TimestampField()
-    updated_at = TimestampField()
+    created_at = serializers.SerializerMethodField()
+    updated_at = serializers.SerializerMethodField()
+
+    def get_created_at(self, instance):
+        return instance.source_created_at.timestamp() if instance.source_created_at else None
```

Comment:
This would do the trick yes, but maybe fallback to created_at just in case would be best